simp_le.py

```diff
@@ -56,7 +56,7 @@
 VERSION = '0'
 URL = 'https://github.com/kuba/simp_le'
 
-LE_STAGING_URI = 'https://acme-staging.api.letsencrypt.org/directory'
+LE_STAGING_URI = 'https://acme-v01.api.letsencrypt.org/directory'
```

Comment:
This constant has intentionally `STAGING` in name... please create a new constants and reference in the the argument setup code.